README.md

```diff
@@ -1,116 +1,170 @@
-This is a readme for the RETNET team
+# Are You Using Retentive Networks? (RetNets) 📝
 
-Make sure to have Python=3.11
-When you use a .sh script, run this to grant permission "chmod +x retnet.sh" for the script you want to use
-Then do this "./retnet.sh" to run the script
+This project focuses on the comparison and analysis of the RetNet vs the Transformer architecture, utilizing Microsoft's TorchScale library for implementation. More information can be found in our paper [Are You Using Retentive Networks?](./301R_Retnet_Paper.pdf).
 
-_Hyperparameters Scaling_
+## Reference to Original Paper
 
-### Python Code Examples
+Our study is based on research detailed in the paper [Retentive Network: A Successor to Transformer for Large Language Models](https://arxiv.org/abs/2307.08621). For more in-depth information and methodology of the RetNet architecture, refer to this paper.
 
-#### 0. **Preparing Data**:
-This repository is designed to be run on a cluster. As such, the data must be downloaded and prepared before running the training script.
+## Base Architecture - Microsoft TorchScale
 
-To download data from the HuggingFace Hub, run the `download_data.py` script with the following parameters:
-- dataset_name : the name of the dataset on HF hub
-- dataset_subset : the subset to use
-- dataset_dir : the name of the folder to save the dataset in. Typically, you can just keep this the same as the dataset_name
+This project is built upon [Microsoft TorchScale](https://github.com/microsoft/torchscale), which provides basic implementations of the two architectures for our research. TorchScale provides a library of foundational architecture implementations for training Transformer-based deep learning models. We have leveraged its capabilities to develop our comparison between RetNets and Transformers.
```

Comment:
between RetNet and Transformer
Perhaps
between the RetNet and Transformer architectures.

Reply:
Distilled.